sdcm/teardown_validators/sstables.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from functools import partial
+
+from sdcm.cluster import BaseNode
+from sdcm.sct_events import Severity
+from sdcm.sct_events.teardown_validators import ValidatorEvent, ScrubValidationErrorEvent
+from sdcm.teardown_validators.base import TeardownValidator
+from sdcm.utils.common import S3Storage, ParallelObject
+from sdcm.utils.s3_remote_uploader import upload_remote_files_directly_to_s3
+
+LOGGER = logging.getLogger(__name__)
+
+
+class SstablesValidator(TeardownValidator):  # pylint: disable=too-few-public-methods
+    validator_name = 'scrub'
+
+    def _upload_corrupted_files(self, node: BaseNode, quarantine_log_lines):
+        # get quarantine dir from lines:
+        # INFO  2024-04-02 12:40:24,787 [shard 0:stre] sstable - Moving sstable /var/lib/scylla/data/system_schema/columns-24101c25a2ae3af787c1b40ee1aca33f/me-3gey_0z3c_2h5vl2ogebmg26ku9t-big-Data.db to "/var/lib/scylla/data/system_schema/columns-24101c25a2ae3af787c1b40ee1aca33f/quarantine"  # pylint: disable=line-too-long
+        # print(log_lines)
+        quarantine_dirs = {line.split(
```

Comment:
Can you please split that long line to the following more readable smaller lines?
```
quarantine_dirs = {
    line.split(' to "')[1][:-2]
    for line in quarantine_log_lines
    if 'sstable - Moving sstable' in line and 'quarantine' in line
}
```

Reply:
fixed